hvpy/api_groups/jpeg2000/tests/test_get_jpx.py

```diff
@@ -8,8 +8,8 @@
 
 def test_raw_response():
     response = getJPX(
-        startTime=datetime(2014, 1, 1, 0, 0, 0),
-        endTime=datetime(2014, 1, 1, 0, 45, 0),
+        startTime=datetime.today() - timedelta(days=1),
```

Comment:
I think we are going to repeat this in every test function, it might be easier to move them to the top of the file instead. Or maybe in fact a pytest fixture.

Reply:
Should I do it for every test file which has datetime hardcoded?

Reply:
Yes please